### ISSUE TITLE
[Partner Aware Onboarding] Disable Woo Payments promotions

### ISSUE THREAD

**Issue by moon0326** [Automattic/wc-calypso-bridge#1422]
Project Thread: pdibGW-2H4-p2
Related Issue: https://github.com/woocommerce/woocommerce/issues/43386

Blocked by: https://github.com/woocommerce/woocommerce/issues/43389

In order to maximize the partner's branding, we should avoid promoting Woo Payments, as it might conflict with the partner's interests. We should disable Woo Payments promotions when the site is created through the partner-aware onboarding flow.

### Acceptance Criteria

- Check the `woocommerce_onboarding_profile.partner` option and disable all Woo Payments promotions when the option is not empty.

**Comment by moon0326** [comment]
I've made a few changes in https://github.com/Automattic/wc-calypso-bridge/pull/1426, but I'm putting this on hold until we have a full list of requirements.
